### PR TITLE
feat(metabase): use instance label for urn component

### PR DESCRIPTION
**Pull request by StewartJingga**
new mandatory config to label extracted metabase metadata.

```
source:
  name: metabase
  config:
    ...
    instance_label: my-metabase
```

this will be used as the urn component like below

```
{
  "urn": "metabase::my-metabase/dashboard/192"
}
```

### CODE REVIEW

File: plugins/extractors/metabase/metabase.go [hunk 1]
```diff
@@ -25,15 +25,17 @@ var summary string
...
 type Config struct {
-	Host      string `mapstructure:"host" validate:"required"`
-	Username  string `mapstructure:"username" validate:"required"`
-	Password  string `mapstructure:"password" validate:"required"`
-	SessionID string `mapstructure:"session_id"`
+	Host          string `mapstructure:"host" validate:"required"`
+	InstanceLabel string `mapstructure:"instance_label" validate:"required"`
+	Username      string `mapstructure:"username" validate:"required"`
+	Password      string `mapstructure:"password" validate:"required"`
+	SessionID     string `mapstructure:"session_id"`
```
Comment (GrayFlash) [inline plugins/extractors/metabase/metabase.go:38]: @StewartJingga with SessionID now in the config, the Username and Password are optional right?? So I think the `validate:"required"` should be removed.

Comment (StewartJingga) [reply plugins/extractors/metabase/metabase.go]: @GrayFlash updated 🙏

**Review by mabdh — APPROVED** [review]

**Review by GrayFlash — APPROVED** [review]

**Review by scortier — APPROVED** [review]

**Comment by GrayFlash** [comment]
@StewartJingga the test for elasticsearch fails in cli as well as my local. Can you check it once??

**Comment by GrayFlash** [comment]
sorry it passed locally, I guess the order in which we get output fields is inconsistent. Shouldn’t be an issue for this PR.

**Pull request merged**